WatchApp Extension/Scenes/GlucoseChartScene.swift

```diff
@@ -334,7 +339,7 @@ class GlucoseChartScene: SKScene {
 
         correctionRange?.quantityBetween(start: start, end: end).forEach({ (range) in
             let (sprite, created) = getSprite(forHash: range.chartHashValue)
-            sprite.color = UIColor.glucose.withAlphaComponent(activeOverride != nil ? 0.2 : 0.3)
+            sprite.color = UIColor.glucose.withAlphaComponent(activeOverride != nil ? 0.3 : 0.5)
```

Comment:
These color ranges are too bright, and drown out the glucose value points.

Reply:
I agree that the color of the target ranges starts to look pretty similar to the color of the glucose points as the alpha value gets larger.  However, I don't think the very low alphas here work well overall.  At 0.2 or 0.3, I find the target ranges very hard to see, especially for narrow ranges.  With the dark background, anything with low alpha looks pretty dark as well, and I think the model that works well on the iPhone app (same color with different alphas) works less well with the small screen and dark background here.  [Calculating contrast ratios](https://gist.github.com/elnjensen/e7b649e83b6213f9fbe8e503d59f98a8) of the blended color (taking the alpha into account and blending with the background), alphas of 0.2 and 0.3 give contrast ratios of 1.5 and 1.8.  Apple's [Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/watchos/visual-design/color/) recommend a minimum of 4.5.   Now, we don't have to hit that for every element, but it does suggest how far below that we are.

A different approach to increasing the contrast but still maintaining a visual distinction between elements is to use a slightly different color for the ranges, but keep the alpha up.  That way the contrast is there, and the visual connection between similar elements is maintained by not having the colors *too* different, but they compete with each other less.  In the original PR we used `UIColor(red: 158/255, green: 215/255, blue: 245/255)` with alphas of 0.4 and 0.6 for disabled and active ranges, respectively.  Combined with the background we're currently using, those have contrast of 2.5 and 3.2, still below 4.5 but noticeably more than the values above.  

This is what that color scheme looks like: 
![img_3028](https://user-images.githubusercontent.com/2545845/46900337-5b66fa80-ce6e-11e8-99cc-9320902ff3b6.JPG)

And this is the color scheme from the code that's currently in dev (glucose color with alpha=0.2/0.3): 
![img_3032](https://user-images.githubusercontent.com/2545845/46900503-3922ac00-ce71-11e8-802a-0bbd78c963c3.JPG)

I'm not wedded to that exact color scheme, but I do think more contrast would be helpful here.

Reply:
The contrast that is important here is between the chart points and the chart ranges.  Both of your suggestions reduce the difference between them.

Reply:
Clearly we don't agree on this, which is fine.  I'll revert the changes and we can see how those colors work in practice.